Tasks/AzureResourceGroupDeploymentV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -7,7 +7,7 @@
   "loc.group.displayName.AzureDetails": "Azure Details",
   "loc.group.displayName.Template": "Template",
   "loc.group.displayName.AdvancedDeploymentOptions": "Advanced deployment options for virtual machines",
-  "loc.group.displayName.Outputs": "Outputs",
+  "loc.group.displayName.Advanced": "Advanced",
```

Comment:
Please validate this change locally once

Reply:
+1